toolkit-test-app/build.gradle

```diff
@@ -70,6 +73,8 @@ dependencies {
     implementation "androidx.constraintlayout:constraintlayout:$droid.constraintLayout"
     implementation "androidx.multidex:multidex:$droid.multiDex"
     implementation "androidx.lifecycle:lifecycle-extensions:$droid.lifecycle"
+    implementation 'androidx.navigation:navigation-fragment-ktx:2.3.4'
+    implementation 'androidx.navigation:navigation-ui-ktx:2.3.4'
```

Comment:
move the versions into `versions.gradle`

Reply:
done